examples/incompressible_flow/2d-backward-facing-step/Bisection.py

```diff
@@ -0,0 +1,96 @@
+"""
+Postprocessing code for 2D-backward-facing-step example
+Computes the reattachment length (x_r) for Re = 100
+for several meshes with a bisection algorithm
+
+Author : Charles Le Pailleur
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import pyvista as pv
+
+########################################
+########################################
+
+# EXAMPLE TO COMPUTE x_r (REATTACHMENT LENGTH) FOR Re = 100 
+# NOTE : THIS FILE MUST BE IN THE "2d-backward-facing-step" DIRECTORY TO 
+#        WORK PROPERLY
+
+# VARIABLES
+L_in = 15                    # Inlet length
+x_r_ref = 2.922              # Benchmark value for Re = 100 (Erturk 2008)
+							 # (see attached .txt file for other Reynolds
+							 #  x_r values)
+x_inf = 2.                   # Initial guesses
+x_sup = 3.
+tol = 1e-12                  # Bisection stop criterion
+
+# DATA EXTRACTION
+n = 11                       # Number of VTU files to be read
+file = list(range(0,n))
+data = list(range(0,n))
+N_elements = np.zeros(n)
+for i in range(0,n):
+    if i<10:
+        file[i] = ('Reynolds100-600/backward_facing_step_output.000' 
+                   + str(i) + '.0000.vtu')
+    elif i>=10:
+        file[i] = ('Reynolds100-600/backward_facing_step_output.00' 
+                   + str(i) + '.0000.vtu')
```

Comment:
In python you can use formating for this, such as : `f'{i:05d}'` (this is just a slight suggestion)

Reply:
Thank you for the info, this solved a LOT of pain in some of my scripts :)

Reply:
:). I learnt something today also ^^!. Lovely